templates/service_user_config_schema.json

```diff
@@ -50,6 +67,35 @@
           "null"
         ]
       },
+      "index_patterns": {
+        "items": {
+          "additionalProperties": false,
+          "description": "Allows you to create glob style patterns and set a max number of indexes matching thispattern you want to keep. Creating indexes exceeding this value will cause the oldest one to getdeleted. You could for example create a pattern looking like 'logs.?' and then create index logs.1,logs.2 etc, it will delete logs.1 once you create logs.6. Note: Setting max_index_count to 0 will donothing and the pattern gets ignored.",
```

Comment:
`s/thispattern/the pattern/`? `getdeleted`. `logs.?` example could include "does not apply to `logs.10`. `donothing`